javascript/code-challenges/treeIntersection/README.md

```diff
@@ -0,0 +1,32 @@
+# Tree Intersection
+
+## Code Challenge 32
+
+Authors: Tahmina Ringer, Jeremy Penning, & Ashley Moore
+
+- Write a function called tree_intersection that takes two binary tree parameters.
+
+- Without utilizing any of the built-in library methods available to your language, return a set of values found in both trees.
+
+- The output array can only have unique numbers
+
+- INPUT: Two Binary Trees
+- OUTPUT:  Common Values in each tree
+
+### Big O
+
+- Space -O(1)
+- Time -O(n)
+
+### Edge Cases
+
+- If no numbers match
+- There is nothing in the the tree
+
+## Approach & Efficiency
+
+I wrote my code first and then wrote tests to validate the code.
+
+### Whiteboard
+
+- [Whiteboard](whiteboard.jpg)
```

Comment:
You are off to a great start, once you have a chance to study this more, get code and tests written, I will regrade for you!